tests/foreman/api/test_classparameters.py

```diff
@@ -0,0 +1,1034 @@
+# -*- encoding: utf-8 -*-
+"""Test class for Smart/Puppet Class Parameter"""
+
+from robottelo.decorators import run_only_on, stubbed, tier1, tier2, tier3
+from robottelo.test import APITestCase
+
+
+class SmartClassParametersTestCase(APITestCase):
+    """Implements Smart Class Parameter tests in API"""
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_list_parameters_by_host_id(self):
+        """List all the parameters included in specific Host by its id.
+
+        @feature: Class Parameters - List
+
+        @assert: Parameters listed for specific Host.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_list_parameters_by_hostgroup_id(self):
+        """List all the parameters included in specific HostGroup by id.
+
+        @feature: Class Parameters - List
+
+        @assert: Parameters listed for specific HostGroup.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_list_parameters_by_puppetclass_id(self):
+        """List all the parameters for specific puppet class by id.
+
+        @feature: Class Parameters - List
+
+        @assert: Parameters listed for specific Puppet class.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_list_parameters_by_environment_id(self):
+        """List all the parameters for specific environment by id.
+
+        @feature: Class Parameters - List
+
+        @assert: Parameters listed for specific environment.
+
+        @status: Manual
+        """
+
+    @run_only_on('sat')
+    @stubbed()
+    @tier3
+    def test_positive_override_checkbox(self):
+        """Override the Default Parameter value.
+
+        @feature: Class Parameters - Override
+
+        @steps:
+
+        1.  Set override to True.
```

Comment:
Separate list index from the content with just one space. Please fix this for all the added stubs.